integrations/ollama/pyproject.toml

```diff
@@ -26,7 +26,7 @@ classifiers = [
     "Programming Language :: Python :: Implementation :: CPython",
     "Programming Language :: Python :: Implementation :: PyPy",
 ]
-dependencies = ["haystack-ai", "ollama>=0.4.0"]
+dependencies = ["haystack-ai", "ollama>=0.4.0", "pydantic>=2.0.0"]
```

Comment:
Ollama python client already depends on `pydantic`, no?
I would not add a direct dependency if not required.